book/2-end/server/server.js

```diff
@@ -1,79 +1,14 @@
-const express = require('express');
```

Comment:
Probably didn't mean to remove this file also?

Reply:
Not sure what happened here either.